### PR TITLE
Fix receiving variables from dev tools integration tests pipeline

### PR DESCRIPTION
**Pull request by Sunnatillo**
Fix for pipeline part is [here](https://github.com/Nordix/metal3-dev-tools/pull/580)

This PR fixes receiving IMAGE_NAME and  IMAGE LOCATION variables from pipeline for metal3 dev tools integration tests. Metal3 dev tools integration tests are not using temporarily built node image, but  default node image. Cause of the problem is IMAGE_NAME and iMAGE_LOCATION variables are not passed to the tests properly. This issue started appearing after we randomized the vars.sh file name [here](https://github.com/metal3-io/project-infra/blob/ef01400cb9c069290865fa01bd7e08b218c84499/jenkins/scripts/integration_test.sh#L129)

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/lgtm
/hold
until the other PR is merged.

**Comment by Sunnatillo** [comment]
Both PRs are tested [here](https://jenkins.nordix.org/job/metal3_dev_tools_integration_test_centos/84/consoleFull)

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**
